confgenerator/self_logs.go

```diff
@@ -108,6 +108,16 @@ func generateFluentBitSelfLogsComponents(ctx context.Context) []fluentbit.Compon
 			},
 		},
 	}.Components(ctx, fluentBitSelfLogsTag, "fluent-bit-self-log-regex-parsing")...)
+	if logLevel == "debug" {
```

Comment:
This would disable completely all fluent-bit `debug` logs. Is there any situation where we would want debug logs ? From my fluent-bit self logs exploration they haven't been useful at all, but there may be some case i'm not aware off.